e2e/test-util.ts

```diff
@@ -226,7 +226,7 @@ export async function createSession(page: Page, sessionName: string) {
   await page
     .getByRole('heading', { name: 'App close' })
     .getByLabel('close')
-    .click();
```

Comment:
15000 ms does not seem to be enough.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lSyr8xXz1wdXALkJKzVx/13e74f58-246a-4d96-9ec0-3363e880f11d.png)